.circleci/config.yml

```diff
@@ -0,0 +1,65 @@
+var_1: &docker_image circleci/node:10.13.0-browsers
+var_2: &cache_key raiden-webui-{{ .Branch }}-{{ checksum "package-lock.json" }}
+
+
+anchor_1: &defaults
+  working_directory: ~/src
+  docker:
+    - image: *docker_image
+
+anchor_2: &root_package_lock_key
+  key: *cache_key
+
+anchor_3: &attach_options
+  at: .
+
+
+version: 2
+jobs:
+  install:
+    <<: *defaults
+    steps:
+      - checkout
+      - restore_cache: *root_package_lock_key
```

Comment:
This is just a suggestion for avoiding future vulnerbilities. 
Ideally restoring and saving the cache should happen right before (restore) and right after (save) the data process of the data containing the cache. 

Also attaching and persisting the workspace should happen at the beginning and at the end respectively. 

```suggestion
steps:
      - checkout
#cache intructions
      - restore_cache: *root_package_lock_key
      - run: npm install
      - save_cache:
          <<: *root_package_lock_key
          paths:
            - "node_modules"
# rest of the steps ..
     - persist_to_workspace:
          root: .
          paths:
            - ./*
```